src/main/java/org/jenkinsci/plugins/workflow/cps/InterpolatedSecretsDetector.java

```diff
@@ -0,0 +1,67 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import hudson.EnvVars;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.jenkinsci.plugins.workflow.cps.view.InterpolatedSecretsAction;
+import org.jenkinsci.plugins.workflow.flow.FlowExecutionOwner;
+import org.jenkinsci.plugins.workflow.steps.EnvironmentExpander;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class InterpolatedSecretsDetector {
+    private EnvVars envVars;
+    private Set<String> watchedVars;
+    private List<String> scanResults;
+    private CpsFlowExecution exec;
```

Comment:
```suggestion
    private final EnvVars envVars;
    private final Set<String> watchedVars;
    private List<String> scanResults;
    private final CpsFlowExecution exec;
```

Reply:
updated in 79d255a8427bc41a575bb6f808b355dfaadecb3e

Reply:
class removed in cc5b3b0678b9607e6cfd197b56b9ed0c4fff8c18